app.rb

```diff
@@ -32,8 +32,8 @@
   # Get basic info on all monitors.
   begin
     monitors = Pingometer.new(PINGOMETER_USER, PINGOMETER_PASS).monitors
-  rescue
-    @error_message = "Our status monitoring system, Pingometer, appears to be having problems."
+  rescue => error
+    @error_message = "Our status monitoring system, Pingometer, appears to be having problems. #{error}"
```

Comment:
I think that’s probably a reasonable assessment. That said, it’s a confusing message to anybody non-expert who might eventually look at this stuff. The other (maybe better) option would be to log the pingometer error (so you’d see it on stderr in dev) instead of putting it in the UI.